recipes/numexpr/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="2.6.1" %}
+
+package:
+    name: numexpr
+    version: {{ version }}
+
+source:
+    fn: numexpr-{{ version}}.tar.gz
+    url: https://github.com/pydata/numexpr/archive/v{{ version }}.tar.gz
+    sha256: e92c83d066fa8da63864d69b5f218287cc31437ae844db77390f2183123aab22
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
```

Comment:
Can you specify what flags are relevant for this case?

I am not adding the `toolchain` in any of the feedstocks I maintain for the following reasons:

- those flags are not needed all the time\*
- the extra flags may cause unexpected issues like the one we faced with `gdal` (ping @kmuehlbauer) making debuging hard due to the unknonwn changes
- the `toolchain` is not versioned in any `feedstock`, but it is constantly updated breaking build reproducibility
- even if we start versioning them we are adding an extra layer that makes it difficult to inspect and third party using our recipes are hurt b/c of that

\* The only cases I see the where some flags are needed is to use `clang` instead of `gcc`. If we have a `gcc` recipe in the future and adopt it everywhere the `toolchain` won't be needed at all. But that is another topic for the future.